packages/universe-components/src/table/DonationsTableSortable.tsx

```
@@ -24,43 +25,103 @@ interface DonationsTableSortableProps {
+  organization?: string
+  amount: number
+  chain?: string
+  storyId?: string
+  image?: string
+  impactScore?: string
+  impactLabel?: string
+  impact?: string
+}
+
+function money(amount:number){
+  return '$'+amount.toFixed(2)
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)